sqlglot/dialects/dialect.py

```diff
@@ -577,7 +577,6 @@ class Dialect(metaclass=_Dialect):
         exp.DataType.Type.DOUBLE: {
             exp.ApproxQuantile,
             exp.Avg,
-            exp.Div,
```

Comment:
Looks like this was no-op? Added [here](https://github.com/tobymao/sqlglot/commit/c6db1240b8481af0003a4bed4225f8e46578f182#diff-0df617c9c55fa6119c2fc262bbffe88e7981595bd333c95a700b031cfa78f8beR168).

Reply:
We have two `exp.Div` annotations, one as a static `Type.DOUBLE` (this one) and one entry in `ANNOTATORS` which is inferred by `_annotate_div(...)` which will overwrite each other; I thought we should keep the latter and enhance `_annotate_div(...)` if needed.